Samples/PDFPackageTest/GO/PDFPackage_test.go

```diff
@@ -36,9 +36,9 @@ func AddPackage(doc PDFDoc, file string, desc string){
     }
     files.Put(&key[0], len(key), fs.GetSDFObj())
     fs.SetDesc(desc)
-    
+
```

Comment:
It's not important but if you can preserve previous line with tab that will be great.